.travis.yml

```diff
@@ -75,13 +79,6 @@ jobs:
         - docker push $BUILD_IMAGE:$TAG
         - echo "docker push done"
 
-addons:
-  artifacts:
```

Comment:
If CT skip for master, no make sense upload CT logs cuz artifacts disabled for PR's